### DISCUSSION TITLE
Running double instances

### DISCUSSION THREAD

**Discussion by lucamarletta** [TriliumNext/Trilium#2460] [Q&A]
Hi Zadam, 
is it possible to run a double instance with the same trilium exec but reading from another set of data?

The variable TRILIUM_DATA_DIR is just one, so I'm worried they will interfere each other.
I could do otherwise, but I'm interested in this possibility is it exists.

Thanks. I love this tool. It's a milestone in my working way.

**Comment by zadam** [comment]
Sure, just set `TRILIUM_DATA_DIR` per individual process. How is that done depends on the operating system, in Linux (and presumably Mac) it's just calling trilium with `TRILIUM_DATA_DIR=your_path ./trilium`
